### ISSUE TITLE
DEV-1: Add data preparation templates for regression

### ISSUE THREAD

**Issue by TetianaKlymchuk** [TetianaKlymchuk/NN-Model-Templates#1]
Data preparation for regression should include:

- _cleaned dataset_(pickle or csv);
- _labels_ (pickle or csv);
- _separate script for loading data_ (reading pickle or csv, split into test set and training set, feature scaling);
- _separate script for preparing data for time series analysis_: for a given raw input sequence choosing a number of time steps and splitting it into samples.

To work on this issue, please use a branch **DEV-1** created from **devel**. When the work is finished - create a PR to the **devel** branch. At the end of each development circles, code from the **devel** branch (if everything is ok) goes to the **master** branch.
